tasks/hidden_service.yml

```diff
@@ -4,7 +4,7 @@
 
 - name: Add the HiddenServiceDir config to torrc
   blockinfile:
-    dest: /etc/tor/torrc
+    dest: "{{ torrc }}"
```

Comment:
I understand now.
inner variables are not part of the role API, but duplicating is not nice either.
this is fragile but I don't know how to do better yet.